### DISCUSSION TITLE
为何

### DISCUSSION THREAD

**Discussion by AshStar-Y** [share121/inter-knot#2221] [其他]
![image](https://github.com/user-attachments/assets/a0197afa-ca68-46fa-813d-0bd0505732cb)
![image](https://github.com/user-attachments/assets/7afdd8cb-73da-4b28-8e50-8b8e51d3d502)
我加了小助手，用不了，为何

**Comment by ArcSurge** [comment]
高版本的油猴脚本需要打开开发者模式才可以使用

**Reply by AshStar-Y** [reply]
ok了，谢谢

**Comment by HKLHaoBin** [comment]
如果你用不了 为什么还能发贴

<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by AshStar-Y** [reply]
因为
![image](https://github.com/user-attachments/assets/8545dca6-fd19-40b7-bde1-561c5068a948)

**Comment by WJK0323** [comment]
加了开发者模式就好了
